### DISCUSSION TITLE
Electrolyzer Unit Model Customizability?

### DISCUSSION THREAD

**Discussion by ommurphy** [watertap-org/watertap#1474] [Q&A] [answered]
Can the Electrolyzer Unit Model be modified to simulate alternative electrolysis processes such as PEM or AWE?

**Comment by hunterbarber** [comment, marked as answer]
The electrolyzer unit model performance is based on specifying the Faradaic performance of the system. This is a simplified 'yield' model in a sense that doesn't consider features of some electrolyzers like multiphase flow. So if you are interested in modeling PEM or AWE based on specifying the stoichiometric conversion of reactions that occur in the anode and cathode then this model can provide the resulting stream compositions and some electrochemical metrics. 

This model does not simulate predicative conversion based on rate laws and electrochemical design. However, the model structure can be used as a skeleton to develop a higher fidelity electrolyzer model if needed.

Thanks for reaching out, and let me know if this has answered your question.
